src/components/auction/OrderPlacement/index.tsx

```diff
@@ -344,6 +358,15 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
 
   return (
     <>
+      <Wrap>
+        <SectionTitle as="h2">
+          {(auctionState === AuctionState.ORDER_PLACING ||
+            auctionState === AuctionState.ORDER_PLACING_AND_CANCELING) &&
+            'Place Order'}
+          {auctionState === AuctionState.CLAIMING && 'Claim Proceeds'}
```

Comment:
This component will never be rendered in the claiming state. No need for the distinguishment here.

Reply:
Indeed this logic needs to move into the claimer component

Reply:
done